export_img.py

```diff
@@ -37,33 +34,21 @@ def find_inkscape_path():
 
 inkscape = find_inkscape_path()
 
-def create_png_directory():
-    """
-    Check if png directory exist, if not create new one
-    """
-    dir = "../png"
-    if os.path.isdir(dir):
-        pass
-    else:
-        os.mkdir(dir)
-
 def export_svg_png():
     """
     Export img.svg to img.png with Inkscape
     """
-    for img in base.glob("*.svg"):
+    for img in Path("./img").glob("**/*.svg"):
         name = str(img)
         parts = []
-        new_name = ""
-
         for part in img.with_suffix(".png").parts:
             if part == "svg":
                 parts.append("png")
             else:
                 parts.append(part)
-            new_name = str(Path(*parts))
-        subprocess.run([inkscape, name, "--export-png=" + "../png/" + new_name, "--export-area-page"], check = True,)
+        new_name = Path(*parts)
+        new_name.parent.mkdir(exist_ok = True, parents= True)
+        subprocess.run([inkscape, name, "--export-png=" + str(new_name), "--export-area-page"], check = True,)
```

Comment:
Toto je pro mne hodne magicke, ale nevim, jestli chtit vysvetlovat kazdy kus kodu komentarem je ok, nebo ne - teoreticky se muzu podivat do dokumentace a zjistit parametry `subprocess.run`...

Reply:
Nebyla jsem na tomto srazu, možná proto je to pro mne tak magické... Ale když se dívám jen na kód, není mi úplně jasné, kde hledat vysvětlení: opět můžu prohlédnout blog nebo googlit, až najdu dokumentaci knihovny. Nebo zkopírovat odkaz, který někdo prozíravě nachystal čtenářům v zakomentovaném řádku :)